### PR TITLE
EPMRPP-90359 || Create redux store for organization

### PR DESCRIPTION
**Pull request by BlazarQSO**
## PR Checklist

* [x] Have you verified that the PR is pointing to the correct target branch? (`develop` for features/bugfixes, other if mentioned in the task)
* [x] Have you verified that your branch is consistent with the target branch and has no conflicts? (if not, make a rebase under the target branch)
* [x] Have you checked that everything works within the branch according to the task description and tested it locally?
* [x] Have you run the linter (`npm run lint`) prior to submission? Enable the git hook on commit in your IDE to run it and format the code automatically.
* [x] Have you run the tests locally and added/updated them if needed?
* [x] Have you checked that app can be built (`npm run build`)?
* [x] Have you made sure that all the necessary pipelines has been successfully completed?
* [x] If the task requires translations to be updated, have you done this by running the `manage:translations` script?

### CODE REVIEW

File: app/src/common/urls.js [hunk 1]
```diff
@@ -127,7 +127,8 @@ export const URLS = {
   apiKeys: (userId) => `${urlCommonBase}users/${userId}/api-keys`,
   apiKeyById: (userId, apiKeyId) => `${urlCommonBase}users/${userId}/api-keys/${apiKeyId}`,
 
-  organizationsList: () => `${urlBase}organizations`,
+  organizationsList: (name) => `${urlBase}organizations${getQueryParams({ name })}`,
+  organizationsById: (organizationId) => `${urlBase}organizations/${organizationId}`,
```
Comment (AmsterGet) [inline app/src/common/urls.js:131]: **Optional**. May be?
```suggestion
  organizationList: (name) => `${urlBase}organizations${getQueryParams({ name })}`,
  organizationById: (organizationId) => `${urlBase}organizations/${organizationId}`,
```

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/controllers/auth/sagas.js [hunk 2]
```diff
@@ -116,7 +117,9 @@ function* loginSuccessHandler({ payload }) {
   yield put(fetchUserAction());
   yield all([take([FETCH_USER_SUCCESS, FETCH_USER_ERROR]), take(SET_ACTIVE_PROJECT_KEY)]);
   const projectKey = yield select(activeProjectKeySelector);
+  yield put(fetchOrganizationsAction());
```
[the remark this answers is not in the document]
Comment (BlazarQSO) [reply app/src/controllers/auth/sagas.js]: Done.

File: app/src/common/urls.js (hunk 1 above)
Comment (BlazarQSO) [reply app/src/common/urls.js]: Done.

File: app/src/controllers/organizations/reducer.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { combineReducers } from 'redux';
+import {} from 'controllers/filter/constants';
```
Comment (AmsterGet) [inline app/src/controllers/organizations/reducer.js:18]: Do we need this?

File: app/src/controllers/organizations/reducer.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      return true;
+    case FETCH_ORGANIZATIONS_SUCCESS:
+      return false;
+    default:
+      return state;
+  }
+};
+
+export const organizationsReducer = combineReducers({
+  info: organizationsInfoReducer,
```
Comment (AmsterGet) [inline app/src/controllers/organizations/reducer.js:49]: I suggest to name it as `list` instead of info to easier understand that it is an array.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/controllers/organizations/reducer.js (hunk 3 above)
Comment (BlazarQSO) [reply app/src/controllers/organizations/reducer.js:18]: Done.

File: app/src/controllers/organizations/reducer.js (hunk 4 above)
Comment (BlazarQSO) [reply app/src/controllers/organizations/reducer.js:49]: Done.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**